style/buttons.less

```diff
@@ -0,0 +1,20 @@
+@btn-border-radius: 4px;
+@primary-background: #f47642;
+@primary-hover-background: #fa804d;
+
+// Override the default text color from bootstrap-material-design
+.btn-primary,
+.btn-success { color: @white !important; }
+
+.btn-primary {
+	font-size: 18px;
```

Comment:
erg, I set the preference in dw to tab= two spaces. Looks like that isn't recognized elsewhere, annoying.